app/commands.py

```diff
@@ -146,3 +147,18 @@ def fix_notification_statuses_not_in_sync(self):
             print('Committed {} updates at {}'.format(len(result), datetime.utcnow()))
             db.session.commit()
             result = db.session.execute(subq_hist).fetchall()
+
+
+class PopulateMonthlyBilling(Command):
+        option_list = (
+            Option('-s', '-service-id', dest='service_id',
+                   help="Service id to populate monthly billing for"),
```

Comment:
The intention of is to start testing it.
After that it might be useful if we feel we need to re-run a month that is not the current month.
But once we are happy this is working we can write one to build all the data for all services. Once we have an idea of how long it will take.